CIP-XXXX/README.md

```diff
@@ -0,0 +1,114 @@
+---
+CIP: ?
+Title: Cardano URIs - CIP-0019 Address Representation
```

Comment:
```suggestion
Title: Cardano URIs - Address Representation
```
It takes some readers & perhaps some CIP title parsers for a ride if another CIP designator is included in a CIP's title.  Given that we don't have a different Cardano address specification — and that further changes to that spec will generally be CIP-0019 updates (or versions) rather than new CIPs — I think it's safer & better to drop the CIP-0019 from the title.